service/cluster/arangodb/machine.go

```diff
@@ -321,19 +322,22 @@ func (c *arangodbCluster) createMachine(index int) (*arangodb, error) {
 			fmt.Sprintf("--starter.join=%s:%d", c.dockerHosts[0].IP, c.MasterPort),
 		)
 	}
+	dockerCertPath := os.Getenv("DOCKER_CERT_PATH")
 	opts := dc.CreateContainerOptions{
 		Name: name,
 		Config: &dc.Config{
 			Image: c.ArangodbConfig.ArangodbImage,
 			Cmd:   args,
 			Tty:   true,
+			Env:   os.Environ(),
```

Comment:
This is a complete pass through of all envs from the client?

Reply:
Yes. And specifically all DOCKER stuff.